packages/paste-website/src/pages/components/data-grid/index.mdx

```diff
@@ -241,6 +241,24 @@ This package provides the following component exports, which correspond closely
 You are free to put any Paste component in the Data Grid cells and it should just work.
 Keep in mind that anything that dynamically updates an element's `tabIndex` value may fail to work correctly.
 
+### Displaying missing data
+
+If the Data Grid can’t show data yet (for example, it’s in a draft state), leave cells blank.
+
+If the data underlying the Data Grid is active but the value for the cell is zero, show 0 in the cell.
+
+If a piece of data is missing because the metric in a certain column does not apply to a given row, show “N/A” and use the HTML abbreviation element to communicate the full meaning of the abbreviation (“Not applicable”). This ensures any customers using assistive technologies will understand the cell entry.
+
+### Errors
+
+Avoid documenting an entire error message within a row. Use an indicator, like an icon or an icon with a short message. Then use other methods, like a popup, side sheet, or full page to communicate complete error and resolution information. 
```

Comment:
```suggestion
Avoid documenting an entire error message within a row. Use an indicator, like an icon or an icon with a short message. Then use other methods, like a [Popover](/components/popover), [Modal](/components/modal), [Side Modal](/components/side-modal), or full page to communicate complete error and resolution information. 
```

Reply:
> or an icon with a short message

Where does the short message live? In the first cell of the row? Or in the popover I get when I click the icon?

Reply:
In Segment at least, it's typically a badge or very short message contained within a cell in the row. If comms or flex frequently shows just the icon, the popover works too. I can make this clearer in the documentation though.